demos/modals/Modals.module.scss

```diff
@@ -168,7 +168,7 @@
 }
 
 .menuContent {
-  padding: 16px 40px 16px 40px;
+  padding: 16px 24px 16px 24px;
```

Comment:
you can keep` padding: 16px 24px`; since thats the same as `padding: 16px 24px 16px 24px;`